.semaphore/semaphore.yml

```diff
@@ -0,0 +1,37 @@
+---
+version: "v1.0"
+name: "Vaultenv CI Pipeline"
+
+agent:
+  machine:
+    type: "e1-standard-2"
+    os_image: "ubuntu1804"
+
+blocks:
+  - name: "Run checks"
+    task:
+      jobs:
+        - name: "Run tests"
+          commands:
+            - "checkout"
+
+            # For some reason, Semaphore CI sets this variable, but it causes the nix installation to fail
+            - unset LD_LIBRARY_PATH
+
+            # Restore `/nix` cache. Create the directory first, otherwise we encounter
+            # permission errors. We do this because the Semaphore cache is faster than
+            # both Cachix and cache.nixos.org.
+            - "sudo mkdir /nix"
+            - "sudo chown -R semaphore:semaphore /nix"
+            - "cache restore nix-store-"
+
+            # Install Nix and source the shell configuration immediately.
+            - "./nix/install"
```

Comment:
Can you check if we're allowed to vendor this script, based on the script's source and the license of this repo?

Reply:
Good call, I'll check

Reply:
The install scripts is from the https://github.com/NixOS/nix repository, which is under LGPL-2.1. 
This is not compatible with BSD-3-Clause, so I'll remove the installation scripts in favor of downloading it and verifying its MD5.